src/toolbars/toolbar-component.js

```diff
@@ -61,6 +62,11 @@ angular.module('patternfly.toolbars').component('pfToolbar', {
       return ctrl.config.viewsConfig && (ctrl.config.viewsConfig.currentView === viewId);
     }
 
+    function isTableViewSelected () {
+      return (ctrl.config.viewsConfig && (ctrl.config.viewsConfig.currentView === 'tableView')) ||
+             ctrl.config.isTableView;
```

Comment:
A bit concerned that a mistake setting isTableView to true when also passing a viewConfig will cause this to return true even if a different view type is selected.